policyengine_us/variables/gov/states/de/tax/income/subtractions/exclusions/de_pre_exclusions_agi.py

```diff
@@ -0,0 +1,13 @@
+from policyengine_us.model_api import *
+
+
+class de_pre_exclusions_agi(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Delaware agi before exclusions"
```

Comment:
Avoid abbreviations in variable labels (parameter labels are OK), and when using them capitalize them; these show up in the app
```suggestion
    label = "Delaware adjusted gross income before exclusions"
```